android/src/main/res/values-night/styles.xml

```diff
@@ -1,9 +1,12 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
+
+	<!-- color from https://m1.material.io/style/color.html#color-themes -->
 	<style name="SpinnerDatePickerDialog" parent="SpinnerDatePickerDialogBase">
-		<item name="colorPrimary">@android:color/white</item>
+		<item name="android:windowBackground">#424242</item>
```

Comment:
nit, Why not referencing colors from `@android:color/`

Reply:
@capezzbr if you tell me what color to reference, I'll do it! 😅 
Maybe I'm blind but I didn't find the right color... and I'm surprised I need to define it, I though the theme would handle it for me...

Reply:
Now that I look at this change closer,
> I thought the theme would handle it for me...
I am thinking the same, why would we need to re-define the window color, I'd assume it should come from the theme

Reply:
Could it be that you don't have to explicitly create `values-night/styles.xml` since `Theme.AppCompat.DayNight.Dialog` will handle that?

Reply:
From the official documentation https://developer.android.com/develop/ui/views/theming/darktheme

> ?android:attr/textColorPrimary This is a general purpose text color. It is near-black in Light theme and near-white in Dark themes. It contains a disabled state.

so in theory that value will change whether the user has enabled dark/light mode, and the app theme should reflect that

Reply:
Yeah Android themes are kinda hard to nail down, I think overriding the color is a good solution for now. tbh I'd have expected the theme to take care of the background color and adapt primary colors too